tests/bugs/9267-autodiff-max-gradient-runner.py

```
@@ -0,0 +1,177 @@
+            positions=p,
+            dims=dims,
+            lossOut=loss,
+        ).launchRaw(blockSize=(THREADS, 1, 1), gridSize=(blocks, 1, 1))
+        return p, loss
+
+    def backward(p, loss):
+        p_copy = p.detach().requires_grad_(True)
+        loss_copy = loss.detach()
+        grad_loss = torch.ones_like(loss_copy)
+        blocks = (n + THREADS - 1) // THREADS
+        kernel = getattr(module, p.grad_fn.__class__.__module__.split(".")[0] if hasattr(p, "grad_fn") else "loss_kernel_vec_max")
```

Ruff (0.15.1)

[error] Local variable `kernel` is assigned to but never used

Remove assignment to unused variable `kernel`

(F841)